docs/metric/README.md

```diff
@@ -1,6 +1,6 @@
 ## Minio Prometheus Metric
 
-Minio server exposes an endpoint for Promethueus to scrape server data at `/minio/metric`.
+Minio server exposes an endpoint for Promethueus to scrape server data at `/minio/prometheus/metric`.
```

Comment:
Typo `/metric` here.

Reply:
@sinhaashish can you address this minor issue ? LGTM otherwise